src/state/research.js

```diff
@@ -0,0 +1,82 @@
+import { cloneDeep, merge } from 'lodash';
+import jobBase from '@/state/jobBase';
+import jobSingleAction from '@/state/jobSingleAction';
+
+import { ACTIONS } from "@/data/research"
+import { RESEARCH_UPGRADE_PERCENT } from "@/data/upgrades";
+
+
+const research = merge(cloneDeep(jobBase), cloneDeep(jobSingleAction), {
+	state: {
+		rndPoint: 0,
```

Comment:
On a similar note as the Research Points thing, once I've moved to a `money`-like item, this needs to be taken out

Reply:
probably not doing this after all